components/fetchCampusLocation/FetchCampusLocation.component.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import {
+  StyleSheet,
+  View,
+  Text,
+  Dimensions,
+  TouchableHighlight
+} from "react-native";
+
+const FetchCampusLocation = props => {
+  /**
+   * this hook sets a boolean variable isSGW to know which current
+   * button is pressed
+   */
+  const [isSGW, setIsSGW] = useState(true);
+
+  /**
+   * Changes the boolean value and the coordinates that will change
+   * the map view
+   */
+  const onSGWPressButton = () => {
+    setIsSGW(true);
+    props.regionChange({
+      latitude: 45.495869,
+      longitude: -73.578107,
+      latitudeDelta: 0.0522,
+      longitudeDelta: 0.0421
+    });
+  };
+
+  const onLoyolaPressButton = () => {
+    setIsSGW(false);
+    props.regionChange({
+      latitude: 45.4584,
+      longitude: -73.64045,
+      latitudeDelta: 0.0522,
+      longitudeDelta: 0.0421
+    });
+  };
+
+  return (
+    <View style={styles.container}>
+      <TouchableHighlight onPress={onSGWPressButton} underlayColor="white">
+        <View
+          style={[isSGW ? styles.buttonSelected : styles.buttonNotSelected]}
+        >
+          <Text
+            style={[
+              isSGW ? styles.buttonTextSelected : styles.buttonTextNotSelected
+            ]}
+          >
+            SGW
+          </Text>
+        </View>
+      </TouchableHighlight>
+      <TouchableHighlight onPress={onLoyolaPressButton} underlayColor="white">
+        <View
+          style={[isSGW ? styles.buttonNotSelected : styles.buttonSelected]}
+        >
+          <Text
+            style={[
+              isSGW ? styles.buttonTextNotSelected : styles.buttonTextSelected
+            ]}
+          >
+            Loyola
+          </Text>
+        </View>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+/**
+ * TODO: styling might need to be changed @Ragith
```

Comment:
Should be addressed before this pr is merged

Reply:
not an issue, but maybe he has a better way to do it